test/workers/generic_worker_test.rb

```diff
@@ -36,9 +36,15 @@ def teardown
     project_media_id = pm.id
     tags_json = ['one', 'two'].to_json
 
-    assert_difference "Tag.where(annotation_type: 'tag').count", difference = 2 do
+    assert_difference "Tag.where(annotation_type: 'tag').count", 2 do
       GenericWorker.perform_async('Tag', 'create_project_media_tags', project_media_id, tags_json, user_id: pm.user_id)
     end
+    tags_json = [''].to_json
```

Comment:
@melsawy just a question: Could this second part (from line 42 to 47) be a separate test?

Reply:
I see that title for testing is `should run a job, without raising an error`  so I append this case instead of creating a new test with team, item, etc...

Reply:
That makes total sense, thank you @melsawy